src/pages/Homepage.jsx

```diff
@@ -1,70 +1,83 @@
-import { useState } from 'react';
+import { useCallback, useState } from 'react';
+import { v4 as uuid } from 'uuid';
 import TimerComponent from '../components/TimerComponent';
 import ActiveTeamsComponent from '../components/ActiveTeamsComponent';
 import EliminatedTeamsComponent from '../components/EliminatedTeamComponent';
+import findTeam from '../utils/index.js';
 
 const Homepage = () => {
   const [teams, setTeams] = useState([]);
   const [draggedIndex, setDraggedIndex] = useState(null);
 
   const handleAddTeam = (type) => {
-    setTeams([
-      ...teams,
+    setTeams((prevState) => [
+      ...prevState,
       {
+        id: uuid(),
         name: '',
         score: 0,
-        active: type === 'active' ? true : false,
+        active: type === 'active',
```

Comment:
`type === 'active'` is already a boolean. No need to return it separately

Reply:
Yeah, I actually though I wrote this, thanks for your review.